test/101_cross_hosts_2_test.sh

```diff
@@ -0,0 +1,50 @@
+#! /bin/bash
+
+. "$(dirname "$0")/config.sh"
+
+C1=10.2.1.4
+C2=10.2.1.7
+UNIVERSE=10.2.0.0/16
+SUBNET_1=10.2.2.0/24
+SUBNET_2=10.2.3.0/24
+IMAGE=weaveworks/network-tester
+
+wait_for_connections() {
+    for i in $(seq 1 45); do
+        if run_on $HOST1 "curl -sS http://127.0.0.1:6784/status | grep \"$SUCCESS\"" ; then
+            return
+        fi
+        echo "Waiting for connections"
+        sleep 1
+    done
+    echo "Timed out waiting for connections to establish" >&2
+    exit 1
+}
+
+wait_for_network_tester_status() {
+    for i in $(seq 1 45); do
+        status=$($SSH $HOST1 curl -sS http://127.0.0.1:8080/status)
```

Comment:
Would it be worth adding a comment explaining that even though we start `network-tester` on two hosts, we only need to wait for one to be ready because they "sync" with other instances internally already?